### PR TITLE
ADD groot normalizer

### PR DESCRIPTION
**Pull request by Vedanth-Ramji**

### CODE REVIEW

**Comment by Vedanth-Ramji** [comment]
I've marked this as ready for review now as the normalizer is out. Once #61 is merged, I'll go ahead and add the tests for groot

**Comment by luispedro** [comment]
There are several unanswered comments (I just added a few more)

**Comment by Vedanth-Ramji** [comment]
Oh, could you please let me know where I could find these comments? I can't find them in my mail or on GitHub.

**Review by luispedro — CHANGES_REQUESTED** [review]

File: db_harmonisation/README.md [hunk 1]
```diff
@@ -76,6 +76,13 @@ The RGI outputs of CDSs & contigs are combined with resfinder & argannot mapping
 
 > Note: as BacMet is now ignored, there will be less megares mappings (compared to when all of megares was run through RGI using `contig` mode) as loose hits from RGI won't be associated with BacMet entries.
 
+# Groot Databases
+
+Groot uses `resfinder`, `CARD`, and `argannot`. However, the versions of `argannot` and `resfinder` used are much older than the default `argannot` and `resfinder`. `CARD` always points to the latest version.
```
[the remark this answers is not in the document]
Comment (Vedanth-Ramji) [reply db_harmonisation/README.md]: The `groot` documentations says that the latest versions of `card`, `argannot`, and `resfinder` are used to create the `groot db`, however, when I took a look at the [code to generate the `groot-db`](https://[raw.githubusercontent.com/will-rowe/groot/master/db/groot-database/make-groot-dbs.sh](https://raw.githubusercontent.com/will-rowe/groot/master/db/groot-database/make-groot-dbs.sh)), I found that only the latest version of the CARD database is downloaded, while older versions of resfinder and argannot (v3) are used - these were the latest versions of `resfinder` and `argannot` when the groot documentation was written.

**Comment by luispedro** [comment]
Thanks. I think this context is not clear from the proposed documentation text.

**Comment by Vedanth-Ramji** [comment]
I just updated the readme for db_harmonisation now. Is it ok now?

**Pull request merged**